auth-web/src/store/modules/user.ts

```diff
@@ -261,7 +261,18 @@ export default class UserModule extends VuexModule {
   public async getUserAccountSettings () {
     const response = await UserService.getUserAccountSettings(this.context.state['userProfile'].keycloakGuid)
     if (response && response.data) {
-      const orgs = response.data.filter(userSettings => (userSettings.type === 'ACCOUNT'))
+      // filter by account type and sort by name
+      const orgs = response.data.filter(userSettings => (userSettings.type === 'ACCOUNT')).sort((a, b) => {
```

Comment:
use localeCompare with sensitivity for checking case insensitive..
also arrow function will make the whole code in one line i think

Reply:
addressed. :)